Source/dvlnet/zerotier_native.cpp

```diff
@@ -9,6 +9,21 @@
 #include "utils/sdl2_backports.h"
 #endif
 
+#if defined(_WIN64) || defined(_WIN32)

```

Comment:
Xbox nxdk currently does not have `std::filesystem` but let's guard against it being added in the future:

```suggestion
#if (defined(_WIN64) || defined(_WIN32)) && !defined(NXDK)
```